timegm conflicts on AIX

## Description
Multiple customers have reported an issue with compiling on AIX, and getting an error about a timegm conflict:

`../config.h:240:8: error: conflicting types for 'timegm'`

## Expected Behavior
Duo Unix should compile on AIX systems

## Actual Behavior
Some AIX systems get the above error

## Steps to Reproduce
Unknown

## Workarounds
One reported workaround is to modify config.h to comment out the timegm handling:
```
/*
#ifndef HAVE_TIMEGM
time_t timegm(struct tm *tm);
#endif
*/
 ```

--- a/acconfig.h
+++ b/acconfig.h
@@ -35,6 +35,7 @@ size_t strnlen(const char *str, size_t maxlen);
 #endif
 
 #ifndef HAVE_TIMEGM
+#include <time.h>
 
 time_t timegm(struct tm *tm);
 #endif

